client/my-sites/upgrades/domain-management/edit/registered-domain.jsx

```diff
@@ -58,8 +58,9 @@ const RegisteredDomain = React.createClass( {
 			transferPath = paths.domainManagementTransfer( slug, name );
 
 		if ( hasPrivacyProtection ) {
+			let label;
 			if ( privateDomain ) {
-				return this.getLabel( {
+				label = this.getLabel( {
```

Comment:
That's why I mentioned the eslint rule and linked to the documentation for it :) The solution is to just use this format:
```
if (stuff) {
   return foo;
}
return bar;
```
instead of
```
if (stuff) {
   return foo;
} else {
   return bar;
}
```